src/Substructure.jl

```diff
@@ -0,0 +1,313 @@
+"""
+    has_parents(p, history) -> (boolean, Int64, Int64)
+
+Checks if the jet `p` is a child of two other jets, after clustering 
+
+# Arguments
+- `p`: The jet to check.
+- `history`: The vector of history element obtained from the cluster sequence after clustering.
+
+# Returns
+- (boolean, Int64, Int64): true or false depending on if the jet has a parent or not. If the jet has a parent, returns the indices of the parent jets in the history element. Otherwise, returns -2 (NonexistentParent).
+"""
+function has_parents(p::PseudoJet, history::Vector{HistoryElement})
```

Comment:
This signature is ok as long as the function is *internal*. Anything the user would call should used a `ClusterSequence` object, so that we can evolve the implementation if we want to.